pkg/osbuild/treeinfo_stage.go

```diff
@@ -0,0 +1,152 @@
+package osbuild
+
+import (
+	"encoding/json"
+
+	"github.com/osbuild/images/pkg/arch"
+)
+
+// treeinfoStageOptions represents the options of the treeinfo stage.
+// All paths are relative to .treeinfo path specified via Path field.
+// See https://github.com/osbuild/osbuild/blob/v165/stages/org.osbuild.treeinfo.meta.json
+type treeinfoStageOptions struct {
```

Comment:
Nitpick: Not exporting this struct makes very little sense, given that all other structs are exported. But in fact, the `NewTreeinfoStage()` does not take any of these types as an argument, so none of them need to be exported. In any case, it would be good to be consistent.